sdks/go/pkg/beam/core/runtime/harness/harness.go

```diff
@@ -546,14 +546,15 @@ func (c *control) handleInstruction(ctx context.Context, req *fnpb.InstructionRe
 			}
 		}
 
-		mons, pylds := monitoring(plan, store, c.runnerCapabilities[URNMonitoringInfoShortID])
+		mons, pylds, ConsumingReceivedData := monitoring(plan, store, c.runnerCapabilities[URNMonitoringInfoShortID])
```

Comment:
Please make this lower case: ConsumingReceivedData. I'm pretty sure it'll trip a linter warning when we import it otherwise.

Reply:
Addressed. Thanks!